frontends/pytorch/test/module_import/annotations/class-annotator-repr.py

```diff
@@ -0,0 +1,74 @@
+# -*- Python -*-
+# This file is licensed under a pytorch-style license
+# See frontends/pytorch/LICENSE for license information.
+
+import typing
+
+import torch
+import torch_mlir
+
+# RUN: %PYTHON %s | FileCheck %s
+
+mb = torch_mlir.ModuleBuilder()
+
+
+class Submodule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+        self.exported = 1
+        self.not_exported = 2
+
+    def forward(self):
+        return self.not_exported_method()
+
+    def not_exported_method(self):
+        return
+
+
+class TestModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+        self.s = Submodule()
+
+    def forward(self):
+        return self.s.forward()
+
+
+test_module = TestModule()
+recursivescriptmodule = torch.jit.script(test_module)
+
+annotator = torch_mlir.ClassAnnotator()
+class_type = recursivescriptmodule._c._type()
+
+annotator.exportNone(class_type)
```

Comment:
There is likely some sugar we can add to this so that it can be effectively used as a decorator on a class type (fine for a followup, or I can fiddle with it since there is a certain pythonic art to it). Fine to do later/easier to do once you have the shape of the thing to make it nice.

Reply:
Woudl appreciate help on the "pythonic art" :)